ror-demo-cluster/conf/es/readonlyrest.yml

```
@@ -61,3 +99,11 @@ readonlyrest:
       groups:
         - id: "EndUsers"
           name: "End Users"
+
+    - username: adm3
+      auth_key: adm3:test
+      groups:
+        - id: "kib_tracy_default_adm_ext98107"
+          name: "kib_tracy_default_adm_ext98107"
+        - id: "kib_tracy_tenant1_adm_ext98107" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)